trade-finance-manager-ui/server/controllers/case/amendments/underwriting/amendment-lead-underwriter/index.js

```diff
@@ -152,14 +131,9 @@ const postAssignAmendmentLeadUnderwriter = async (req, res) => {
     return res.redirect(`/case/${dealId}/underwriting`);
   }
 
-  const underWriterUserToAdd = await api.getUser(assignedToValue);
-
   const update = {
     leadUnderwriter: {
```

Comment:
can we not having this as `leadUnderwriterId: '123'`?

Reply:
done